packages/design/src/TextArea/TextArea.story.tsx

```diff
@@ -0,0 +1,35 @@
+/**
+Copyright 2022 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React from 'react';
+import { TextArea } from './TextArea';
+
+export default {
+  title: 'Design/TextArea',
+};
+
+export const TextAreas = () => (
+  <>
+    <TextArea mb={4} placeholder="Enter Some long text" />
+    <TextArea mb={4} hasError={true} defaultValue="This field has an error" />
+    <TextArea mb={4} autoFocus defaultValue="This field is focused" />
```

Comment:
It seems that the `autoFocus` is not passed to the actual `<textarea>` tag, at least in the storybook the `≤textarea>` doesn't seem to have `autofocus` attribute.

Reply:
It's weird, in a regular app it works, so probably it is something related to `Storybook`.

I will remove these examples (also in `Input.story`) as they are not working.